client/devdocs/controller.js

```diff
@@ -90,9 +87,7 @@ const devdocs = {
 
 	// UI components
 	design: function( context, next ) {
-		context.primary = React.createElement( DesignAssetsComponent, {
-			component: context.params.component,
-		} );
+		context.primary = <AsyncLoad component={ context.params.component } count={ 65 } require="./design" />;
```

Comment:
I don't think we need 65 placeholders. Having 3 (or a bit more to fill a page) should be enough as we just want to show something is being loaded. The component will also be loaded quickly before user is able to scroll through all 65 of them. ;) Lastly, we wouldn't need to increase this number each time when adding a new component and make the code a bit simpler.

Reply:
Actually 5 seems to work pretty well and makes a laptop-sized screen fill the page with placeholders, which gives off the right effect (as most of the pages are quite long).

Pushed an update where count is always 5.